### DISCUSSION TITLE
Migration to unRAID

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [jlesage/docker-nginx-proxy-manager#301] [General]
Hi, I have working NPM with few domains (duckDNS, Dynu and no-ip) on Dietpi (Debian) home server. Now I want to migrate to my new machine with unRAID server. I stop old docker instance and install new also jlesage image in unRAID. Can somebody help me how to do it wright? What folders and files to copy? Or maybe add domains from clean install?
Because I copy config folders to unraid before and nothing work for me. I just change IP and ports.
Regards.

**Comment by jlesage** [comment]
All data used by the container is stored at `/mnt/user/appdata/NginxProxyManager`.  If you copy this folder to the new server and create a container with the same setting, you should be good to go.
Port forwarding on your router should obviously be updated accordingly.
